app/scripts/components/LocationMap.js

```diff
@@ -1,9 +1,9 @@
 import L from 'leaflet'
 import PropTypes from 'prop-types'
 import React, { Component } from 'react'
-import { Map, TileLayer, Marker } from 'react-leaflet'
+import { Map, TileLayer, Marker, Popup } from 'react-leaflet'
 
-const DEFAULT_ZOOM = 13
+const DEFAULT_ZOOM = 11
```

Comment:
This looks like a good default for London :smile: Maybe we should introduce a config variable here, what do you think?

Reply:
yeh, good idea! Or I was thinking it could just start at "world view" and people zoom ;-)